### ISSUE TITLE
Doesn't work at latest nightly rust toolchain

### ISSUE THREAD

**Issue by CertainLach** [dylanmckay/protocol#5]
```rust
error[E0271]: type mismatch resolving `<i64 as std::convert::TryFrom<i8>>::Error == std::num::TryFromIntError`
   --> ...\protocol-0.3.1\src\primitives\numerics.rs:102:6
    |
102 | impl Integer for i64 { }
    |      ^^^^^^^ expected enum `std::convert::Infallible`, found struct `std::num::TryFromIntError`
    |
    = note: expected type `std::convert::Infallible`
               found type `std::num::TryFromIntError`
```

**Comment by CertainLach** [comment]
@dylanmckay, can you publish updated crate on crates.io?

**Comment by dylanmckay** [comment]
Published [0.3.2](https://crates.io/crates/protocol) 👌
